Events description page layout design completed

FangChat opened the pull request:
**THE EVENTS DESCRIPTION PAGE LAYOUT DESIGN HAS BEEN COMPLETED**

**Changes made:**
1. Created a new component "EventDescriptionBox" to dynamically load data and show the description of the event.
2. Created two new routes for the description page of the aero modelling and block chain workshop events.

**Design of the layout on PCs / Laptops / Tablets**:

![EventsDescriptionPage_laptop](https://github.com/cittakshashila/frontend/assets/90413824/61431c28-4f56-4434-96b0-78ac61bb1876)

**Design of the layout on Phones:**


![EventsDescriptionPage_mobile1](https://github.com/cittakshashila/frontend/assets/90413824/d2e22c23-17d8-41a8-a5ec-07d4ca3fe288)





![EventsDescriptionPage_mobile2](https://github.com/cittakshashila/frontend/assets/90413824/1e73abc3-b0c2-474e-9088-3caa80e74f39)



**Features added:**
1. Event description page layout design

MKMukeshkannan reviewed (changes requested):
Design:
- Add hover effects on buttons.
- tablet view is not looking good, have the mobile design for tablet sizes too. like the design should only change above _lg_ not _md_

Code:
- Improve semantics by using _section_ instead of _div_
- Reduce arbitrary values,
- Reduce viewport height and width,

FangChat commented on src/components/EventDescriptionBox.tsx:
```diff
@@ -0,0 +1,204 @@
+
+      {/*Right side div*/}
+      <div className="flex flex-col w-[70%] h-[100%] gap-[2.2vw] justify-center">
+        <div className="block">
```
Changes made:
1. Create a new component EventButton that accepts classnames for more customization
2. Changed components/index.tsx to export EventButton
3. Completely removed the use of arbitrary values in EventDescriptionBox.tsx
4. Refractored the code in EventDescriptionBox.tsx to reduce nesting of divs
5. Added custom colors and textUnderlineOffset size in tailwind.config.js
6. Added svg icon files in assets and replaced the instagram post, share link and download buttons in the PC view with the icons

Updated PC layout design:

![image](https://github.com/cittakshashila/frontend/assets/90413824/cdb715c3-648c-4229-9114-d6b8a476a2ee)


Updated mobile layout design:

![EventsDescriptionPage_mobile1](https://github.com/cittakshashila/frontend/assets/90413824/1e851ac6-1b43-4987-8865-501e4c5cdcbc)





![EventsDescriptionPage_mobile2](https://github.com/cittakshashila/frontend/assets/90413824/03bc193e-b0e6-4de2-b69a-d1845bb4c20c)





![EventsDescriptionPage_mobile3](https://github.com/cittakshashila/frontend/assets/90413824/cdadec7a-9731-4521-9735-f89e14d7ddac)

FangChat commented:
Changes made:
1. Create a new component EventButton that accepts classnames for more customization
2. Changed components/index.tsx to export EventButton
3. Completely removed the use of arbitrary values in EventDescriptionBox.tsx
4. Refractored the code in EventDescriptionBox.tsx to reduce nesting of divs
5. Added custom colors and textUnderlineOffset size in tailwind.config.js
6. Added svg icon files in assets and replaced the instagram post, share link and download buttons in the PC view with the icons

Updated PC layout design:

![image](https://github.com/cittakshashila/frontend/assets/90413824/cdb715c3-648c-4229-9114-d6b8a476a2ee)


Updated mobile layout design:

![EventsDescriptionPage_mobile1](https://github.com/cittakshashila/frontend/assets/90413824/1e851ac6-1b43-4987-8865-501e4c5cdcbc)





![EventsDescriptionPage_mobile2](https://github.com/cittakshashila/frontend/assets/90413824/03bc193e-b0e6-4de2-b69a-d1845bb4c20c)





![EventsDescriptionPage_mobile3](https://github.com/cittakshashila/frontend/assets/90413824/cdadec7a-9731-4521-9735-f89e14d7ddac)
